app/models/grda_warehouse/client_match.rb

```diff
@@ -177,6 +205,7 @@ def score_contribution(fields)
     end
 
     def accept!(user: nil)
+      user ||= User.system_user
```

Comment:
I think you can just do this:
```suggestion
    def accept!(user: User.system_user)
```
And then the API defines the behavior if you leave it out.